### PR TITLE
ENH: nuclear norm

### PR DESCRIPTION
**Pull request by argriffing**
Yet another matrix norm.

### CODE REVIEW

File: numpy/linalg/linalg.py [hunk 1]
```diff
@@ -1980,6 +1981,8 @@ def norm(x, ord=None, axis=None, keepdims=False):
 
         :math:`||A||_F = [\\sum_{i,j} abs(a_{i,j})^2]^{1/2}`
 
+    The nuclear norm is the sum of singular values.
```
Comment (charris) [inline numpy/linalg/linalg.py]: "the" singular values.

**Comment by charris** [comment]
LGTM modulo nitpick.

**Comment by argriffing** [comment]
fixed the docstring grammar

**Comment by charris** [comment]
Thanks Alex.

**Pull request merged**